### PR TITLE
XDU_Smart_Belt_Based_On_Neural_Network

### PR DESCRIPTION
**Pull request by Tu9711**
队伍名称：划水八天
作品名称：基于神经网络的智能腰带

### CODE REVIEW

**Comment by YuguoWH** [comment]
Please remove 5121f5aa256083057ca2f6b6a554b2c5a1918c4f so we could merge your Pull Request

File: ilight_smartdevice/src/FreeRTOSConfig.h [hunk 1]
```diff
@@ -15,7 +15,7 @@
      *    manual, or both from: http://www.FreeRTOS.org/Documentation        *
      *                                                                       *
      *    Thank you!                                                         *
-     *                                                                       *
+     *                                                                       *ppppppppppppppppppppp
```
Comment (YuguoWH) [inline ilight_smartdevice/src/FreeRTOSConfig.h:18]: Please remove this

**Review by YuguoWH — CHANGES_REQUESTED** [review]

**Comment by YuguoWH** [comment]
Hi @Tu9711 , thanks for your new fix. However using a new commit would leave the modified history in our repo which is not pretty and somehow confusing for future review. Could you please checkout to your first commit 859be7e locally and then force push to your `Tu9711:master` branch? 
The commands would probably looks like this: 
```
git checkout 859be7e
git push -f HEAD:master
```
Thank you.

**Comment by Tu9711** [comment]
Hi @YuguoWH thanks for your help , however I have some problems when committing branch changes so i have started a new pull request. Could you please check it ? thank you  #78

**Comment by YuguoWH** [comment]
#78 merged. Thank you.

**Pull request closed without merging**